Source/JavaScriptCore/b3/air/AirValidate.cpp

```diff
@@ -91,9 +91,14 @@ class Validater {
 
                 // forEachArg must return Arg&'s that point into the args array.
                 inst.forEachArg(
-                    [&] (Arg& arg, Arg::Role, Bank, Width) {
+                    [&] (Arg& arg, Arg::Role role, Bank, Width width) {
                         VALIDATE(&arg >= &inst.args[0], ("At ", arg, " in ", inst, " in ", *block));
                         VALIDATE(&arg <= &inst.args.last(), ("At ", arg, " in ", inst, " in ", *block));
+
+                        // FIXME: replace with a check for wasm simd instructions.
+                        VALIDATE(Options::useWebAssemblySIMD()
```

Comment:
Is this correct? (When useWebAssemblySIMD is enabled, this validation is skipped).

Reply:
Yes. Unless SIMD is enabled, no instruction can USE more than 64 bits